chart/kubeapps/templates/kubeapps-frontend-config.yaml

```diff
@@ -79,7 +76,7 @@ data:
         proxy_pass http://kubeapps-internal-pinniped-proxy.{{ $.Release.Namespace }}:{{ $.Values.pinnipedProxy.service.port }};
       {{- else }}
         # Otherwise we route directly through to the clusters with existing credentials.
-        proxy_pass {{ default "https://kubernetes.default" .apiServiceURL }};
+        proxy_pass {{ default "https://kubernetes.default" .apiServiceURL }}/$1;
```

Comment:
By putting the "extra" match group into the `proxy_pass` directive I expected to preserve existing behavior of this location and avoid [this error](https://stackoverflow.com/questions/53353572/proxy-pass-cannot-have-uri-part-in-location-given-by-regular-expression).

Reply:
If you look at the [answer to your link](https://stackoverflow.com/a/53354944), you'll see that what we've used is exactly what the answer suggests as their (2) option - straight location with a rewrite and `break`. The error you mention seems, from the question, to stem from not including a `break` in the rewrite, but that's just from a glance.

Reply:
@absoludity right but I just cannot do `proxy_pass http://my-host.com/k8s` inside this location block (due to the trailing `/k8s` AFAIUI).

Reply:
Yes, the solution I've linked to, similar to our existing config, doesn't try to append paths to the `proxy_path` directive, but rather uses a rewrite to set the path.